src/Console/ModelsCommand.php

```diff
@@ -830,7 +837,7 @@ public function setMethod($name, $type = '', $arguments = [], $comment = '')
 
     public function unsetMethod($name)
     {
-        unset($this->methods[strtolower($name)]);
+        unset($this->methods[$name]);
```

Comment:
Not sure if we should change this…

It's from the initial implementation https://github.com/barryvdh/laravel-ide-helper/pull/1198/files#diff-2df8ee218526540f71334c651990edfbfe5345cc53a521b0259c874ff24abd9aR766

ping @jenga201 if you remember why?